### PR TITLE
[sync] 2020/10/5

### PR DESCRIPTION
**Pull request by chenglu**
ref #374

### CODE REVIEW

File: src/docs/deployment/android.md [hunk 1]
```diff
@@ -423,7 +423,7 @@ flag to `flutter build apk` or `flutter build appbundle`.
   Obfuscation and minification can considerably extend compile time
   of the Android application.
   
-  混淆和压缩会大大地延长安卓应用程序的编译时间。
+  混���和压缩会大大地延长安卓应用程序的编译时间。
```
Comment (AlexV525) [inline src/docs/deployment/android.md:426]: Encoding issues?

File: src/docs/deployment/android.md [hunk 2]
```diff
@@ -482,21 +482,36 @@ values in the `defaultConfig` block:
   and the version number display string. You can do this by setting
   the `version` property in the pubspec.yaml file. Consult the version
   information guidance in the [versions documentation][versions].
-
+
```
Comment (AlexV525) [inline src/docs/deployment/android.md:485]: nit: Extra space

File: src/docs/deployment/android.md [hunk 3]
```diff
@@ -690,7 +705,7 @@ native binaries that are not applicable to their device's architecture.
 
 Follow these steps to install the APK on a connected Android device.
 
-按照如下这些步骤，将前一步中构建出来的 APK 安装到 Android 设备上。
+按照如下这些步骤，将前一步中构建出来�� APK 安装到 Android 设备上。
```
Comment (AlexV525) [inline src/docs/deployment/android.md:708]: Encoding x2

File: src/docs/deployment/android.md [hunk 4]
```diff
@@ -800,7 +815,7 @@ rebuild the Flutter app.
 在项目根目录下运行 `flutter pub get`，
 或者使用 IDE 中的 **Pub get** 按钮。
 这将会更新 `local.properties` 文件中的 `versionName` 和 `versionCode`，
-之后它会在你构建 Flutter 应用的时候更新 `build.gradle`。
+之后它会在你构建 Flutter 应用的时候��新 `build.gradle`。
```
Comment (AlexV525) [inline src/docs/deployment/android.md:818]: Encoding x3

File: src/docs/deployment/android.md [hunk 5]
```diff
@@ -886,7 +901,7 @@ in the main menu. Select any of the variants in the **Build Variants**
 panel (debug is the default):
 
 接下来，选择构建变体。在主菜单中点击 **Build > Select Build Variant**。
-从 **Build Variants** 面板中选择任意一个变体 （默认是 debug）。
+从 **Build Variants** 面板中��择任意一个变体 （默认是 debug）。
```
Comment (AlexV525) [inline src/docs/deployment/android.md:904]: Encoding x4

**Review by AlexV525 — CHANGES_REQUESTED** [review]
Several issues were found. 😢

File: src/docs/deployment/android.md (hunk 5 above)
Comment (chenglu) [reply src/docs/deployment/android.md:904]: ok, on my way...

**Review by AlexV525 — APPROVED** [review]
image

**Comment by chenglu** [comment]
@AlexV525 would like to request your LGTM again, thanks for your time!

**Pull request merged**